### PR TITLE
Refactor/set leaflet maps as bg for homepage

### PR DESCRIPTION
**Pull request by Somiel**
Homapage background refactoring, leaflet maps implementation.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -87,3 +87,6 @@ gem 'activeadmin', github: 'activeadmin'
 
 # Use whenever gem for automated users cleanup
 gem 'whenever', :require => false
+
+# Use leaflet for main page background
+gem 'leaflet-rails'
```
Comment (murderface13) [inline Gemfile]: :no_mouth:

Comment (Somiel) [inline Gemfile]: just a transitional )

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -14,4 +14,6 @@
 //= require materialize-sprockets
 //= require jquery_ujs
 //= require turbolinks
-//= require ./orders
\ No newline at end of file
+//= require leaflet
+//= require ./orders
+//= require ./home
```
Comment (murderface13) [inline app/assets/javascripts/application.js]: :no_mouth:

Comment (Somiel) [inline app/assets/javascripts/application.js]: meh..

**Comment by murderface13** [comment]
:+1:

**Pull request merged**